packages/hooks/src/useIsPaymentAgentVisibleInOnboarding.ts

```diff
@@ -0,0 +1,9 @@
+import useAllPaymentAgentList from './useAllPaymentAgentList';
+
+const useIsPaymentAgentVisibleInOnboarding = () => {
```

Comment:
@heorhi-deriv I think this hook is kinda redundant, We can just use the `usePaymentAgentList` hook and check the length wherever we use it 🤔

For example, if we use both `usePaymentAgentList` and `useIsPaymentAgentVisibleInOnboarding` on the same page we are making 2 calls to the socket which one of them is unnecessary 🤔